File: src/main/java/com/dv/telegram/google/SheetData.java
package com.dv.telegram.google;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class SheetData {

    private List<RowData> rows;

    public SheetData() {
        rows = new ArrayList<>();
    }

    public void addRow(RowData row) {
        rows.add(row);
    }
}

File: src/main/java/com/dv/telegram/google/SheetData.kt
package com.dv.telegram.google

class SheetData {
    val rows: MutableList<RowData> = mutableListOf()

    fun addRow(row: RowData) = rows.add(row)

    fun getRowsWithoutFirstRow() = rows.subList(1, rows.size)
}
